nin/backend/compress.js

```diff
@@ -92,8 +94,14 @@ function compress(projectPath, payload, callback) {
           'demo=bootstrap({layers:layers, camerapaths:camerapaths, onprogress: ONPROGRESS, oncomplete: ONCOMPLETE});' +
         '}' +
       '}' +
-    '}</script><canvas class=hide height='+ height + ' width=' + width + '></canvas><img src=# onload=z()><!--';
+    '}</script>' +
```

Comment:
Ok, I wasn't sure if it was just a formatting thing or there was more meaning to it.